packages/cmf/src/cmfConnect.md

```diff
@@ -242,7 +242,16 @@ export default function* myDeLorean() {
 If you have multiple instance of the same component those api support `id` as a second argument.
 
 ```javascript
-const componentState = Clock.getState(state, 'a-component-id');
+import Clock from './Clock.connect';
```

Comment:
can you provide a concrete example on why we need this feature and how to use it

Reply:
If i understand, the feature already exists, having access to componentId in sagas but by the way of component props.
```javascript
myDeLorean({ componentId, props }) {
```
`props.componentId` should be equal to the new `componentId` in most cases unless intentional mind fuck...

And now we can access and rely on a more stable information.

But what of the `default` componentId ? which is `undefined` via component props if not explicitly set. Is it a real 'default' string in the new version or undefined ? i.e. componentId in CMF would be super great to have a consistant string type, at least in the app point of view (external interface of CMF)

Reply:
about the componentId I would like it to be only a props, but it can be set as a function in cmfConnect.
Yet another breaking change...